src/account/useEmailSignIn.ts

```diff
@@ -8,7 +8,10 @@ type TRequest = {
   email: string,
   password: string,
 }
-
+/**
+ * Create email session
+ * {@link https://appwrite.io/docs/client/account?sdk=web-default#accountCreateEmailSession | create email session documentation}
```

Comment:
This is currently how it appears in VS Code.

![](https://i.imgur.com/pSEzRc5.png)

It should look like this (I'm using another function as an example).

![](https://i.imgur.com/EcewNjs.png)

https://github.com/sanny-io/react-appwrite/blob/3f160484efaaf71395a1d30181f1ea5b8d4b7069/src/databases/useDocumentUpdate.ts#L16

Reply:
Alright, got it 👌